google-cloud-spanner/src/test/java/com/google/cloud/spanner/IntegrationTestWithClosedSessionsEnv.java

```diff
@@ -50,7 +50,8 @@ DatabaseClientImpl createDatabaseClient(
         String clientId,
         SessionPool pool,
         boolean useMultiplexedSessionBlindWriteIgnore,
-        MultiplexedSessionDatabaseClient ignore) {
+        MultiplexedSessionDatabaseClient ignore,
+        boolean useMultiplexedSessionForRW) {
```

Comment:
Although this argument is unused, it is necessary because we're overriding the method from SpannerImpl for testing purposes. This is same as the other arguments such as `MultiplexedSessionDatabaseClient ignore` which are getting ignored.